src/Kernel/Kernel.csproj

```diff
@@ -29,10 +29,11 @@
   <ItemGroup>
     <EmbeddedResource Include="res\kernel.js" />
     <EmbeddedResource Include="res\bundle.js" />
+    <EmbeddedResource Include="node_modules\chart.js\dist\Chart.bundle.min.js" Link="res\chart.js" />
     <Content Include="tsconfig.json">
       <CopyToOutputDirectory>Never</CopyToOutputDirectory>
     </Content>
-    <PackageReference Include="Microsoft.TypeScript.MSBuild" Version="3.7.4">
+    <PackageReference Include="Microsoft.TypeScript.MSBuild" Version="3.9.6">
```

Comment:
What was this version change for?

Reply:
This was needed to get the new `import type` statement.